packages/mint-components/src/components/sqm-referral-table/columns/sqm-referral-table-date-column.tsx

```diff
@@ -3,12 +3,29 @@ import { Component, h, Host, Method, Prop } from "@stencil/core";
 import { useRequestRerender } from "../re-render";
 import { ReferralTableColumn } from "./ReferralTableColumn";
 
+/**
+ * @uiName Referral Table Date Column
+ */
 @Component({
   tag: "sqm-referral-table-date-column",
   shadow: true,
 })
 export class ReferralTableDateColumn implements ReferralTableColumn {
+  /**
+   * @uiName Date Column Title
+   */
   @Prop() columnTitle: string = "Date Converted";
+  /**
+   * @uiName Date Displayed
+   * @uiEnum ["dateConverted",
+   * "dateReferralStarted",
+   * "dateFraudChecksCompleted",
+   * "dateModerated",
+   * "dateModified",
+   * "dateReferralEnded",
+   * "dateReferralPaid",
+   * "dateUserModified"]
+   */
   @Prop() dateShown: string = "dateConverted";
```

Comment:
Could update the typescript type to be explicit `"dateConverted" |  "dateReferralStarted" | ...`